AnaTutorial/src/AnaTutorial.h

```diff
@@ -0,0 +1,189 @@
+#ifndef ANATUTORIAL_H__
+#define ANATUTORIAL_H__
+
+#include <fun4all/SubsysReco.h>
+
+/// Class declarations for use in the analysis module
+class Fun4AllHistoManager;
+class PHCompositeNode;
+class TFile;
+class TTree;
+class TH1;
+class TH2;
+class PHCompositeNode;
+class RawClusterContainer;
+class RawCluster;
+class SvtxTrackMap;
+class JetMap;
+class GlobalVertex;
+class PHHepMCGenEventMap;
+class JetEvalStack;
+class JetRecoEval;
+class SvtxTrackEval;
+class PHG4TruthInfoContainer;
+class PHHepMCGenEvent;
+class CaloTriggerInfo;
+class JetTruthEval;
+class SvtxEvalStack;
+
+/// Definition of this analysis module class
+class AnaTutorial : public SubsysReco
+{
+ public:
+  /// Constructor
+  AnaTutorial(const std::string &name = "AnaTutorial",
+              const std::string &fname = "AnaTutorial.root");
+
+  // Destructor
+  virtual ~AnaTutorial();
+
+  /// SubsysReco initialize processing method
+  int Init(PHCompositeNode *);
+
+  /// SubsysReco event processing method
+  int process_event(PHCompositeNode *);
+
+  /// SubsysReco end processing method
+  int End(PHCompositeNode *);
+
+  /// Set the minimum jet pT to cut on
+  void setMinJetPt(float minjetpt) { _minjetpt = minjetpt; }
+
+  /// Set the minimum cluster pT to cut on
+  void setMinClusPt(float mincluspt) { _mincluspt = mincluspt; }
+
+  /// Set things to analyze
+  void analyzeTracks(bool analyzeTracks) { _analyzeTracks = analyzeTracks; }
+  void analyzeClusters(bool analyzeClusters) { _analyzeClusters = analyzeClusters; }
+  void analyzeJets(bool analyzeJets) { _analyzeJets = analyzeJets; }
+  void analyzeTruth(bool analyzeTruth) { _analyzeTruth = analyzeTruth; }
+
+ protected:
+  /// String to contain the outfile name containing the trees
+  std::string outfilename;
```

Comment:
Coding convention 2.2 Use prex `m_` for private/protected data members of classes. [data-member-naming] and not to start with `_`

Reply:
Changed all member variables to have prefix `m_`